template.yaml

```diff
@@ -113,13 +113,30 @@ Resources:
               - method.request.querystring.page
               - method.request.querystring.results
 
+  NvaCristinCreateProjectsFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: cristin-project
+      Handler: no.unit.nva.cristin.projects.CreateCristinProjectHandler::handleRequest
+      Runtime: java11
+      MemorySize: 512
+      Role: !GetAtt ReadCristinAuthenticationRole.Arn
+      Events:
+        NvaCristinProjectsEvent:
+          Type: Api # More info about API Event Source: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#api
+          Properties:
+            RestApiId: !Ref NvaCristinProxyApi
+            Path: /project/
+            Method: post
+
   NvaCristinOneProjectFunction:
     Type: AWS::Serverless::Function
     Properties:
       CodeUri: cristin-project
       Handler: no.unit.nva.cristin.projects.FetchOneCristinProject::handleRequest
       Runtime: java11
       MemorySize: 512
+      Role: !GetAtt ReadCristinAuthenticationRole.Arn
```

Comment:
Why do you need this? Using authenticated client on an open endpoint?

Reply:
Need authentication to read unpublised and/or incomplete projects - authentication is only used on get, not for query